src/test/scala/mesosphere/marathon/api/validation/AppDefinitionValidatorTest.scala

```diff
@@ -325,6 +325,30 @@ class AppDefinitionValidatorTest extends MarathonSpec with Matchers with GivenWh
     assert(validate(container).isFailure)
   }
 
+  test("persistent volume with container path '.' is invalid") {
```

Comment:
the current validation would allow `".hidden"` so I would test that as well to make it obvious